### DISCUSSION TITLE
How to add access control to admin view tabs

### DISCUSSION THREAD

**Discussion by cpaschen** [joomengine/Joomla-Component-Builder#763] [Q&A]
I have an admin view created.
We have created some additional tabs (using Admin View >> Settings >> Tabs)

However, we need to set permissions (ACL groups) on when tabs are displayed for these tabs.
Additionally, we need to set permissions on the 'Publishing' tab as well (we don't want everyone seeing that tab).

I know how I can do this manually in the code to restrict by groups, but is there a way within the JCB GUI to set permissions on these tabs?
(I know that there are ways to add permissions for Admin CUSTOM tabs, but not for the 'regular' admin tabs)

**Comment by Llewellynvdm** [comment]
Looking at [this code](https://github.com/Llewellynvdm/Joomla-Sermon-Distributor/blob/master/admin/views/sermon/tmpl/edit.php#L107) you will see that permissions are added to the publish tab, and so if the user can't edit the created_by, created, state fields, nor delete the item then the publish tab is hidden.

But with the other tabs we have not added any permissions control, and yet I think it is a good feature request.

Please explain more on how you would do this manually with code examples and let me see what I can do. I have done it my  self a few time, mostly on the front-end. Since I would simply remove the whole tap from the front-end view. So I can think how we can do this... but will be helpful to see how you achieve the same.
